contracts/stellar-axelar-operators/src/migrate.rs

```diff
@@ -0,0 +1,25 @@
+use soroban_sdk::{Address, Env, Vec};
+
+mod storage {
+    use soroban_sdk::Address;
+    use stellar_axelar_std::contractstorage;
+
+    #[contractstorage]
+    #[derive(Clone, Debug)]
+    pub enum MigrationDataKey {
+        #[instance]
+        #[status]
+        Operators { account: Address },
+    }
+}
+
+pub type MigrationData = Vec<Address>;
+
+pub fn migrate(env: &Env, migration_data: MigrationData) {
+    for account in migration_data {
+        if storage::is_operators(env, account.clone()) {
+            super::storage::set_operator_status(env, account.clone());
+            storage::remove_operators_status(env, account);
+        }
```

Comment:
```suggestion
        ensure!(storage::is_operators(env, account.clone()), Error::MigrationFailed);

        storage::set_operator_status(env, account.clone());
        legacy_storage::remove_operators_status(env, account);
```
fail if account is not an operator instead since we should the owner should only be migrating known operators